db/system_keyspace.cc

```diff
@@ -262,6 +262,24 @@ schema_ptr raft_snapshots() {
     return schema;
 }
 
+schema_ptr repair() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, REPAIR);
+        return schema_builder(NAME, REPAIR, std::optional(id))
+            .with_column("table_uuid", uuid_type, column_kind::partition_key)
+            .with_column("range_start", utf8_type, column_kind::partition_key)
+            .with_column("range_end", utf8_type, column_kind::partition_key)
+            .with_column("repair_uuid", uuid_type, column_kind::clustering_key)
+            .with_column("repair_time", timestamp_type, column_kind::clustering_key)
+            .with_column("keyspace_name", utf8_type)
+            .with_column("table_name", utf8_type)
```

Comment:
These can be static.

Reply:
What if the name changed later. I want to record the history not just for the tombstone removal.

Reply:
Does it merit duplicating the name for every repair, and for every range in the repair? We should be able to observe renaes in the schema history table.

Reply:
One has to cross check the time frame in the schema history to figure it out.  Did you mean this table

```
cqlsh> SELECT * from system.scylla_table_schema_history ;

 cf_id | schema_version | column_name | clustering_order | column_name_bytes | kind | position | type
-------+----------------+-------------+------------------+-------------------+------+----------+------
```
This has no time info, so we do not know when the name is changed. It  does not have keyspace and name info neither.

Reply:
Actuallly, table names cannot be changed yet. If/when we add the ability to change the table name, it will be reflected in the schema history table.